src/Extensions/Logging/Internal/OmexLoggerProvider.cs

```diff
@@ -0,0 +1,30 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT license.
+
+using Microsoft.Extensions.Logging;
+
+namespace Microsoft.Omex.Extensions.Logging
+{
+	internal class OmexLoggerProvider : ILoggerProvider, ISupportExternalScope
+	{
+		public OmexLoggerProvider(ILogEventSender logsEventSender, IExternalScopeProvider defaultExternalScopeProvider)
+		{
+			m_logsEventSender = logsEventSender;
+			m_defaultExternalScopeProvider = defaultExternalScopeProvider;
+		}
+
+
+		public ILogger CreateLogger(string categoryName) => new OmexLogger(m_logsEventSender, m_externalScopeProvider ?? m_defaultExternalScopeProvider, categoryName);
+
+
+		public void Dispose() { }
```

Comment:
Just to confirm: are we sure we don't want to dispose of anything here? Do any of the fields inherit from `IDisposable`? If so, we should dispose of them here to prevent leaks of unmanaged objects.

Reply:
It's the third time we are discussing this topic :) sorry for the enormous PR size.  We are getting `IDisposable` from `ILoggerProvider` and don't need to use it here.